docs/conf/portland/2025/cfp.rst

```diff
@@ -75,15 +71,11 @@ You are of course welcome to write the talk with other people, as long as only o
 
 **Mix of roles and perspectives**
 
-Talks from the scientific community, fiction writers, system administrators, and support staff – in addition to technical writers and software developers – are all valuable to our attendees.
-
-**Diverse audience**
-
-Likewise, we love the diversity of our community, and we encourage support-folks to attend talks about programming, developers to attend talks about writing, writers to attend talks about data, and so on and so forth.
+Talks from the scientific community, fiction writers, system administrators, and support staff – as well as technical writers and software developers – all are valuable to our attendees.
 
 **Past speakers**
 
-Past speaker roles include but are not limited to
+View our past speaker roles, and always remember, you do not have to fit one of these to be a speaker!
 
 * Writers
 * Developers
```

Comment:
```suggestion
* Developers

You do not have to have one of these roles to be a speaker, we're always adding to this list!
```

Reply:
Feels weird to even have a list?

Reply:
yeah, I'm happy to kill it. The intent was to make sure that folks that aren't tech writers are comfortable submitting talks